raft/log_unstable_test.go

```diff
@@ -98,19 +97,17 @@ func TestMaybeLastIndex(t *testing.T) {
 	}
 
 	for i, tt := range tests {
-		u := unstable{
-			entries:  tt.entries,
-			offset:   tt.offset,
-			snapshot: tt.snap,
-			logger:   raftLogger,
-		}
-		index, ok := u.maybeLastIndex()
-		if ok != tt.wok {
-			t.Errorf("#%d: ok = %t, want %t", i, ok, tt.wok)
-		}
-		if index != tt.windex {
-			t.Errorf("#%d: index = %d, want %d", i, index, tt.windex)
-		}
+		t.Run(fmt.Sprint(i), func(t *testing.T) {
```

Comment:
Please add `tt:=tt` to avoid this theoretical problem
(https://deepsource.io/blog/common-mistakes-in-go/#4-possibly-undesired-value-being-used-in-goroutine).

[It would become real if we called t.Parallel() to speed up the tests.]

Reply:
(the same comment for the other loops as well)

Reply:
In that case, we need to check all the existing test cases in this repo; of course in separate PRs. It might not be an urgent task, because there is no any usage of `t.Parallel()` for now, and we are always cautious on `t.Parallel()`

Reply:
so we don't need to add `tt:=tt`,right?

Reply:
Please update this PR per @ptabor 's comment. And we can revisit all other test cases in this repo in separate PRs.

Reply:
@falser101 Please update this PR per @ptabor 's comment above https://github.com/etcd-io/etcd/pull/14795#discussion_r1025366181